frontend/src/components/Navigator/components/SideNav.css

```diff
@@ -92,4 +92,19 @@
 
 .smart-hub--navigator-item:last-child .smart-hub--navigator-link-active {
   border-bottom-right-radius: 4px;
+}
+
+.alert-fade {
```

Comment:
I personally feel like it could be slightly longer, but that would be more of a question for the demo